include/ts/apidefs.h.in

```diff
@@ -554,6 +554,11 @@ typedef enum {
   TS_EVENT_SSL_CLIENT_HELLO  = 60207,
   TS_EVENT_SSL_SECRET        = 60208,
 
+  TS_EVENT_FILE_CREATED = 60300,
+  TS_EVENT_FILE_UPDATED = 60301,
+  TS_EVENT_FILE_DELETED = 60302,
+  TS_EVENT_FILE_IGNORED = 60303,
+
   TS_EVENT_MGMT_UPDATE = 60300
```

Comment:
Just checking: is it intentional that `TS_EVENT_FILE_CREATED` and `TS_EVENT_MGMT_UPDATE` have the same enum value?

Reply:
Definitely not.  Thanks for catching that.  I've moved file events so they no longer conflict.